### PR TITLE
ADD support for docker 19 and catch all for future versions > 19

### PR DESCRIPTION
**Pull request by mzaneri**

### CODE REVIEW

File: install_scripts/templates/docker-install/19-03-ce.sh [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+#   $ sh test-docker.sh
+#
+# NOTE: Make sure to verify the contents of the script
+#       you downloaded matches the contents of install.sh
+#       located at https://github.com/docker/docker-install
+#       before executing.
+#
+# Git commit from https://github.com/docker/docker-install when
+# the script was uploaded (Should only be modified by upload job):
+SCRIPT_COMMIT_SHA="UNKNOWN"
```
Comment (emosbaugh) [inline install_scripts/templates/docker-install/19-03-ce.sh]: can you add a sha here from the docker repo? https://github.com/docker/docker-install/commit/b50fac9805c3d95fb5e8b16d57b19d333e972e20

File: install_scripts/templates/docker-install/19-03-ce.sh [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+			esac
+		;;
+
+		centos)
+			if [ -z "$dist_version" ] && [ -r /etc/os-release ]; then
+				dist_version="$(. /etc/os-release && echo "$VERSION_ID")"
+			fi
+		;;
+
+		rhel|ol|sles)
```
Comment (emosbaugh) [inline install_scripts/templates/docker-install/19-03-ce.sh]: i feel like you need check_ce_on_ee before this line

Comment (mzaneri) [reply install_scripts/templates/docker-install/19-03-ce.sh]: does line 355 not satisfy this?

Comment (emosbaugh) [reply install_scripts/templates/docker-install/19-03-ce.sh]: Well it will exit on 339 before 355 is ever executed

Comment (mzaneri) [reply install_scripts/templates/docker-install/19-03-ce.sh]: changed

File: install_scripts/templates/docker-install/19-03-ce.sh (hunk 1 above)
Comment (mzaneri) [reply install_scripts/templates/docker-install/19-03-ce.sh]: changed

**Pull request merged**